lib/tbot/config/destination_memory.go

```diff
@@ -28,6 +29,8 @@ const DestinationMemoryType = "memory"
 // DestinationMemory is a memory certificate Destination
 type DestinationMemory struct {
 	store map[string][]byte `yaml:"-"`
+	// mutex protects store in case other routines want to read its content
+	mutex *sync.RWMutex
```

Comment:
If we don't use a pointer to the mutex, we don't need to initialize it in CheckAndSetDefaults